src/pe_mailer/message.py

```diff
@@ -8,6 +8,12 @@
 import logging
 import os.path
 
+logging.basicConfig(
+    filename="pe_reports_Logging.log",
```

Comment:
This filename looks odd to me with a capital "L" - if you're ok with changing it throughout this PR, we typically use all lowercase for filenames:
```suggestion
    filename="pe_reports_logging.log",
```

Reply:
Also, have you considered using a variable to store the filename?  That way if it ever needs to be changed, you only have to change it in one place.

Reply:
Capital "L" in filename addressed in https://github.com/cisagov/pe-reports/pull/169/commits/42c8eada7449166b582ba05c650561490b8e51c7.

@cduhn17 - Any thoughts on this?
> have you considered using a variable to store the filename? That way if it ever needs to be changed, you only have to change it in one place.

Reply:
@dav3r 
I have addressed the log file name and put it in a variable per commit [1881313]([1881313](https://github.com/cisagov/pe-reports/pull/169/commits/1881313ac8d0ba99f29cb122d2ddfa07f4ccfc09))